### PR TITLE
Added PXELinux chain iPXE template

### PR DESCRIPTION
**Pull request by lzap**
I also updated

http://projects.theforeman.org/projects/foreman/wiki/Fetch_boot_files_via_http_instead_of_TFTP

### CODE REVIEW

**Comment by lzap** [comment]
I've fixed the tests broken by https://github.com/theforeman/community-templates/pull/219

File: pxe/PXELinux_chain_iPXE.erb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+<%#
+Chainboots iPXE/gPXE from PXELinux. Make sure the OS has iPXE template
+associated and file ipxe.lkrn was copied in the TFTP directory (e.g. from
+/usr/share/ipxe/ipxe.lkrn or iPXE project web site).
+%>
+
+DEFAULT linux
+LABEL linux
+KERNEL ipxe.lkrn
+APPEND dhcp && chain <%= foreman_url('iPXE') %>
```
Comment (lzap) [inline pxe/PXELinux_chain_iPXE.erb:14]: Oh this needs to be escaped.

Comment (lzap) [inline pxe/PXELinux_chain_iPXE.erb:14]: Both Vim and github shows red, I am escaping, strange.

Comment (lzap) [inline pxe/PXELinux_chain_iPXE.erb:14]: Oh thats because it assumes HTML. Ok.

**Comment by mmoll** [comment]
please rebase after the merge of #222

**Comment by lzap** [comment]
Done.

File: test/test_helper.rb [hunk 2]
```diff
@@ -81,6 +81,7 @@ def ks_console(*args)
   def foreman_url(*args)
     'http://localhost'
   end
+
```
Comment (mmoll) [inline test/test_helper.rb]: please remove this extra line, then we're good to go :)

**Comment by mmoll** [comment]
Works for me :+1:

In theory, we could add support to install and copy the files also for iPXE like we do for TFTP in -installer.

**Comment by lzap** [comment]
Done.

**Comment by mmoll** [comment]
merged, díky @lzap!

**Pull request merged**